### ISSUE TITLE
Digital inputs are not displayed

### ISSUE THREAD

**Issue by CKoneCK** [nielstron/ha_blnet#13]
is it a bug or a feature - is there a workaround?
I read that you are no longer developing it...

**Comment by nielstron** [comment]
Yeah this is an open issue in `pyblnet` (https://github.com/nielstron/pyblnet/issues/39)...
